<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProprioceptionAssessWindow</class>
 <widget class="QMainWindow" name="ProprioceptionAssessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>751</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>751</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PLUTO Proprioception Assessment Control Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>PLUTO Proprioception Assessment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="hocGraph"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbStartStopProtocol">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start Protocol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxPauseProtocol">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pause Protocol</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textInformation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>729</width>
         <height>71</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>729</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Mono</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 255, 0);
background-color: rgb(0, 0, 0);
font: 8pt &quot;Cascadia Mono&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
